baseclasses/BaseRegTest.py

```diff
@@ -126,7 +126,7 @@ def root_add_dict(self, name, d, **kwargs):
         """
         with multi_proc_exception_check(self.comm):
             if self.rank == 0:
-                self._add_dict(name, d, **kwargs)
+                self._add_dict(name, d, name, **kwargs)
```

Comment:
Just making sure I understand this:
`name` is passed twice here because you added `full_name` to the `_add_dict` inputs. When `_add_dict` is called recursively, `full_name` is actually useful. Is that right?

Reply:
That is correct. It would give the full path of the keys that's causing problems. Something like `funcsSens: cruise_cl: cruise_alpha` or something like that. Before, the error message would just say `cruise_alpha` and you have no idea which exact key is causing the problem, since everything's in one nested dict.